server/middlewares/validators/tripValidate.js

```diff
@@ -1,69 +1,98 @@
 /* eslint-disable consistent-return */
 import dotenv from 'dotenv';
-import { tripValidateSchema } from '../../helpers/validateSchema';
+import { tripValidateSchema,tripUpdateValidateSchema } from '../../helpers/validateSchema';
 import cityService from '../../services/cityService';
 import Util from '../../helpers/util';
 import { errorLogger }from '../../helpers/loggerHandle';
-import errorTemp from '../../services/templates/TripsErrorTemplate';
+import {joiErrorsTemplate, cityErrorTemplate} from '../../services/templates/TripsErrorTemplate';
 
 const util = new Util();
-
 dotenv.config();
+class TripValidations{
 
-
- export const tripValidation = async(req, res, next)=>{
-    
+ static async CreateTripJoiValidation(req, res, next){
```

Comment:
I believe, according to JavaScript convention, Class methods should be named starting with a lowercase letter... i.e. this method should be `createTripJoiValidation(req, res, next) ...`

Please do this for other Class methods in your PR.